comm.cpp

```diff
@@ -76,7 +76,8 @@ struct _packed_ mavlink_router_mavlink1_header {
 };
 
 Endpoint::Endpoint(const char *name)
-    : _name{name}
+    : _name{name},
+    _last_packet_len{0}
```

Comment:
use c++11 initialization in the header

Initialization list here is only for fields that are initialized according to the parameters.

And if you were to use it, then the comma should align with ":":

```c++
 Endpoint::Endpoint(const char *name)
     : _name{name}
     , _last_packet_len{0}
```